grpc-examples/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>stargate</artifactId>
         <groupId>io.stargate</groupId>
-        <version>1.0.43-SNAPSHOT</version>
```

Comment:
What's the deal here?

Reply:
I think it's because `grpc-examples` is only in the modules for the default profile and not the `dse` profile.

Reply:
So we need to manually fix this to `1.0.44-SNAPSHOT`?